newsfragments/41736.significant.rst

```diff
@@ -3,5 +3,5 @@ Removed deprecated parameters from core-operators.
 Parameters removed:
 
 - airflow.operators.datetime.BranchDateTimeOperator: use_task_execution_date
-- airflow.operators.trigger_dagrun.TriggerDagRunOperator: execution_date
+- airflow.providers.standard.operators.trigger_dagrun.TriggerDagRunOperator: execution_date
```

Comment:
This change needs to be reverted

Reply:
Thanks for pointing that out, I've reverted the change.